fastlane/Fastfile

```diff
@@ -448,22 +444,16 @@ platform :ios do
       export_options: { method: 'enterprise', iCloudContainerEnvironment: 'Production' }
     )
 
-    sh('mv ../build/WooCommerce.ipa "../build/WooCommerce Alpha.ipa"')
-
-    # NOTE: "ipa" parameter is deprecated in appcenter_upload 1.6.0, but there's a bug in the action that
-    # makes the default gym output override the "file" parameter.
     appcenter_upload(
       api_token: get_required_env('APPCENTER_API_TOKEN'),
       owner_name: 'automattic',
       owner_type: 'organization',
       app_name: 'WooCommerce-Installable-Builds',
-      ipa: 'build/WooCommerce Alpha.ipa',
       destinations: 'All-users-of-WooCommerce-Installable-Builds',
       notify_testers: false
     )
 
-    # Install SentryCLI prior to trying to upload dSYMs
-    sh('curl -sL https://sentry.io/get-cli/ | bash')
+    ensure_sentry_installed
```

Comment:
Unrelated with this PR, but fresh in my memory.

We almost never need to run these lane locally, but when we do, they're a massive time sink. I did this for the WordPress 19.1 finalization and 19.2 code freeze and, sure enough, it didn't succeed on the first time because my Sentry version was not the one the plugin expected. 

We should move these checks at the start of the lane. Fail early.

Ideally, I'd open PRs for these, but I'm dropping this comment here just so it's not only in my brain. The diff reminded me of that.

Reply:
Great call – I've moved this to the top of the lane in 12d75efea48816c1e0c0a776ae22c26a166a0a05